tests/scripts/test_scripts.py

```diff
@@ -1010,32 +1014,67 @@ def test_analyze_gather_tb(tmpdir: str):
     assert run.result["n_tb_dirs"] == 2
 
 
-def test_convert_trajs(tmpdir: str):
-    """Tests that convert_trajs is idempotent and does not change the data."""
-    shutil.copy(OLD_FMT_ROLLOUT_TEST_DATA_PATH, tmpdir)
-    tmp_path = os.path.join(tmpdir, os.path.basename(OLD_FMT_ROLLOUT_TEST_DATA_PATH))
-    with open(tmp_path, "rb") as f:
-        pickle.load(f)  # check it's in pickle format to start with
-    args = ["convert_trajs.py", tmp_path]
-    with mock.patch.object(sys, "argv", args):
-        convert_trajs.main()
+def test_pickle_fmt_rollout_test_data_is_pickle():
+    # WHEN
```

Comment:
Are we doing GWT for all tests now?

Reply:
Yes.

I think it makes tests a lot easier to grasp when reading them again.
It also pushes us to only test one concept at a time in each test. This makes it a lot easier to find the root cause when a test fails.